src/App.jsx

```diff
@@ -1,5 +1,36 @@
 import 'bulma/css/bulma.css';
+import cn from 'classnames';
+
 import './App.scss';
+import { useState } from 'react';
+
+const SORT_ALPHABETICALLY = 'alpabet sort';
+const SORT_BY_LENGTH = 'length sort';
```

Comment:
tip:
it will be better to combine such kind sort types in common object
because  theoretically, the number of types can increase
It is also more convenient to keep constants in a separate file
```suggestion
export const SORT_TYPE = {
  ALPHABETICALLY: 'alphabet',
  LENGTH: 'length',
};
```